CHANGELOG.md

```diff
@@ -1,126 +1,432 @@
 # CHANGELOG
 Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [Unreleased]
+## [2.4.0]
+
 ### 💥 Breaking Changes
 
-### Deprecations
+None
+
+
+### 🗑 Deprecations
+
+None
+
 
 ### 🛡 Security
 
+- Bump makelogs to remove dependency on got ([#2801](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2801))
+- Bump moment-timezone from 0.5.34 to 0.5.37 ([#2361](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2361))
+- Bump percy-agent to use non-beta version ([#2415](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2415))
+- Resolve sub-dependent d3-color version and potential security issue ([#2454](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2454))
```

Comment:
[GMS-2022-4708] Resolve sub-dependent d3-color version and potential security issue